src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure CDN Management Library</Description>
-		<Version>4.0.0-preview</Version>
+		<Version>5.0.0-preview</Version>
```

Comment:
if you are in preview, bumping up major version does not make sense
Also add < packageReleaseNotes > for your changes. It should mention what is changed or what new functionality has been added. If you have multi line text use [CData xml tag inside < packageReleaseNotes >